coopr-docs/docs/source/rest/metrics.rst

```diff
@@ -81,3 +83,149 @@ Example
            "total": 1
        }
    }
+
+
+
+Get Node Usage Metrics
+======================
+
+To retrieve node usage metrics, issue an HTTP GET request to the URI:
+::
+
+ /metrics/nodes/usage
+
+Only admins are allowed to get node usage metrics. Tenant admins will see metrics for their own nodes,
+whereas the superadmin will get metrics for all tenant nodes.
+
+
+HTTP Responses
+==============
+
+The response will be a JSON Object with start and end time as keys, and the node usage metrics as the values.
+node usage metrics include the key-value pairs entered as parameters, followed by the resulting node hour usage
+based on the selected criteria.
+
+.. list-table::
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Status Code
+     - Description
+   * - 200 (OK)
+     - If update was successful
+   * - 403 (FORBIDDEN)
+     - If the user is forbidden from getting node usage metrics.
```

Comment:
I have yet to see a scenario where I get a 403 returned.  200, 401 (missing headers), or 405.